package.json

```diff
@@ -52,11 +52,11 @@
     "cy": "CYPRESS_dexUrl=https://$INGRESS_HOST:$PORT_HTTPS CYPRESS_baseUrl=http://localhost:9999 cypress open",
     "cy:dev": "source ../monitor-ci/.env && CYPRESS_dexUrl=CLOUD CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{cloud,shared}/**/*.*'",
     "cy:dev-oss": "source ../monitor-ci/.env && CYPRESS_dexUrl=OSS CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{oss,shared}/**/*.*'",
-    "generate": "export SHA=a29e24cb19c002132e978d78e6f39bee74ebec56 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
+    "generate": "export SHA=8e8ba92e4764cd8914284315fb87c1945c757497 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
     "generate-local": "export REMOTE=../openapi/ && yarn generate-meta",
     "generate-meta": "if [ -z \"${CLOUD_URL}\" ]; then yarn generate-meta-oss; else yarn generate-meta-cloud; fi",
     "generate-meta-oss": "yarn oss-api && yarn notebooks && yarn unity && yarn managedFunctions && yarn annotations-oss",
-    "generate-meta-cloud": "yarn cloud-api && yarn notebooks && yarn unity && yarn managedFunctions && yarn annotations && yarn cloudPriv",
+    "generate-meta-cloud": "yarn cloud-api && yarn notebooks && yarn unity && yarn managedFunctions && yarn annotations && yarn cloudPriv && yarn pinned",
```

Comment:
godspeed. Just checking this won't be in OSS right?

Reply:
nope